.gitignore

```diff
@@ -3,4 +3,5 @@ bin/configlet.exe
 build/
 **/*.out
 **/*.o
+.*.swp
```

Comment:
```suggestion

```
Can we have this as a separate PR and keep this one related to this exercise only?